beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -419,7 +410,7 @@ def benchmark_resample(
 #
 
 metadata = torchaudio.info(SAMPLE_WAV_PATH)
-print_metadata(metadata, src=SAMPLE_WAV_PATH)
+print(metadata)
```

Comment:
do you also want to add the lines for printing source for the all ones that originally had `src=SOME_PATH`? currently some have it and some don't

Reply:
It might look random but I made intentional decision to remove the source from some of them. I did not quickly come up with a nice way to print source, and I thought the tutorial is readable enough without them.